activerecord/test/cases/base_test.rb

```diff
@@ -885,10 +885,17 @@ def test_dup_of_saved_object_marks_as_dirty_only_changed_attributes
 
   def test_bignum
     company = Company.find(1)
-    company.rating = 2147483647
+    company.rating = 2147483648
     company.save
     company = Company.find(1)
-    assert_equal 2147483647, company.rating
+    assert_equal 2147483648, company.rating
+  end
+
+  unless current_adapter?(:SQLite3Adapter)
```

Comment:
This is due to #22594 (should be fixed by #28379). SQLite3 can store bigint value to PK column, but current integer type interrupts it.